support-figma/extended-layout-plugin/src/localization-module.ts

```diff
@@ -263,65 +336,138 @@ async function localizeNodeAsync(
 }
 
 async function localizeTextNodeAsync(
+  node: TextNode,
+  stringResourceMap: Map<string, StringResource>,
+  staleTextNodes: Array<TextNode>,
+  newTextNodes: Array<TextNode>
+) {
+  var preferredName = getResName(node);
+  if (preferredName) {
+    let normalizedText = normalizeTextNode(node);
+    const isNodeExcluded = isExplicitlyExcluded(node);
+    var cachedCharacters = node.getPluginData(
+      STRING_RES_CHARACTERS_PLUGIN_DATA_KEY
+    );
+    // This node has been exported as string resource before and it hasn't changed,
+    // use its res name and put in the string res map.
+    if (cachedCharacters === JSON.stringify(normalizedText)) {
```

Comment:
I'm not sure if `JSON.stringify` is stable (I think the serialization order is based on when fields were added to an object chronologically, not the alphabetical order of fields -- there might be a flag to control this?).

Reply:
This `normalizedText` is a string or a string array. So I think it should be stable here.